docs/topics/testing/tools.txt

```diff
@@ -112,15 +112,20 @@ Making requests
 
 Use the ``django.test.Client`` class to make requests.
 
-.. class:: Client(enforce_csrf_checks=False, raise_request_exception=True, json_encoder=DjangoJSONEncoder, **defaults)
+.. class:: Client(enforce_csrf_checks=False, raise_request_exception=True, json_encoder=DjangoJSONEncoder, headers=None, **defaults)
```

Comment:
Don't forget to make the `headers` parameter keyword-only everywhere in the docs after following through on Adam's request.